### ISSUE TITLE
Implement Error Metric Selection

### ISSUE THREAD

**Issue by menuka94** [Project-Sustain/validation-service#28]
- [x] PyTorch
- [x] Tensorflow
- [x] ScikitLearn
